Client/mods/deathmatch/logic/CClientBuildingManager.cpp

```diff
@@ -98,9 +98,22 @@ void CClientBuildingManager::DestroyAllForABit()
 
 void CClientBuildingManager::RestoreDestroyed()
 {
-    for (CClientBuilding* building : GetBuildings())
+    bool hasInvalidLods = true;
+    while (hasInvalidLods)
```

Comment:
How many iterations does it usually takes? Isn't it possible that this loop become infinite in some conditions(due to model loading failure for example)?

Reply:
I expect 2 iterations:
1. Generic buildings
2. Lod's

yeah, there is one situation, where `CClientBuilding::Create` doesn't work.

Reply:
Then maybe limit it by adding a maximum number of iterations?

Reply:
No, i'll delete invalid elements

Reply:
> No, i'll delete invalid elements
@TheNormalnij Do you mean you'll add a follow up commit for that?

Reply:
yes

Reply:
fixed

Reply:
> fixed

@TheNormalnij
I don't see a commit when you said "fixed" ? Forgot to push it to your PR?

Reply:
Yes, you are right. Thank you!